setup.py

```diff
@@ -185,7 +188,7 @@ def run(self):
     url                           = URL,
     install_requires              = REQUIRED,
     python_requires               = REQUIRES_PYTHON,
-    include_package_date          = True,
+    # include_package_date          = True,
```

Comment:
I commented this line because I was getting a warning that it's no longer a valid option for distutils. Consider removing it, if its indeed not needed.

Reply:
This can certainly be removed